src/components/CourseCard.tsx

```diff
@@ -13,10 +13,10 @@ export const CourseCard = ({
   onClick: () => void;
 }) => {
   const router = useRouter();
-  const imageUrl = 'banner_placeholder.png';
-  // course.imageUrl && course.imageUrl !== ''
-  //   ? course.imageUrl
-  //   : 'banner_placeholder.png';
+  const imageUrl =
+    course.imageUrl && course.imageUrl !== ''
```

Comment:
`course.imageUrl !== ''`
it's not required as `course.imageUrl` will take care of all falsy values including an empty string